dbservice/postgresdb/dbparam.go

```diff
@@ -0,0 +1,53 @@
+package postgresdb
+
+import (
+	"errors"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+)
+
+var (
```

Comment:
the vars are not specific to postgres. They could be stored in some common package

Reply:
done